src/api/spec/components/notification_action_description_component_spec.rb

```diff
@@ -1,3 +1,5 @@
+require 'decision' # To break the circular dependency between Decision and DecisionFavored and DecisionCleared
+
```

Comment:
Nitpick: shouldn't we include these lines into the last commit, with all the changes of the specs?

Reply:
It should go closer to where the change really is: the split of `Decision` into `DecisionFavored` and `DecisionCleared`, so I think it's good where it is now.